django/applications/catmaid/control/similarity.py

```diff
@@ -427,7 +429,7 @@ def add_from_raw_data(self, project_id, user_id, name, data,
         scoring = data
 
         return NblastConfig.objects.create(project_id=project_id,
-            user=user, name=name, status='complete',
+            user=user, name=name, status='complete', # FIXME: user is not defined
```

Comment:
`user` should be `user_id`, which is passed in as an argument. This function isn't used presently.